src/libraries/System.Security.Cryptography.Pkcs/tests/SignatureSupport.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Pkcs.Tests
+{
+    public class SignatureSupport
+    {
+        private static int _supportsRsaSha1Signatures = 0;
+
+        public static bool SupportsRsaSha1Signatures
```

Comment:
I'll update this and https://github.com/dotnet/runtime/pull/69999 based on the @vcsjones' feedback on the other PR.